hydradx/model/init_utils.py

```diff
@@ -1,100 +1,47 @@
-import copy
-import string
-
 from . import actions
 from .amm import amm
+from .amm import omnipool_amm as oamm
 import ipdb
 
-def complete_initial_values(values: dict, agent_d: dict = None) -> dict:
-    state = amm.initialize_state(values, values['token_list'], agent_d)
-    state['token_list'] = values['token_list']
-    state['L'] = values['L'] if 'L' in values else 0
-    state['fee_assets'] = values['fee_assets']
-    state['fee_LRNA'] = values['fee_LRNA']
-    if 'burn_rate' in values:
-        state['burn_rate'] = values['burn_rate']
-    return state
-
 
 def get_configuration(config_d: dict) -> tuple:
```

Comment:
this may change once we are figuring out how to properly allow notebooks to easily set up either Omnipool or Basilisk AMMs

Reply:
(but for now I support just are doing the translation here as it is instead of trying to adjust all the notebooks a lot)